src/lib/sidebar/pipeline/TimingForm.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+  import { NumberInput, Radio } from "lib/govuk";
+  import type { PipelineTiming } from "types";
+
+  export let data: PipelineTiming;
+  // Are status and timescale required?
```

Comment:
This is a new difference; the fields aren't required at per-feature level, but are at scheme level